services/print/controller/controller.go

```diff
@@ -0,0 +1,33 @@
+package controller
+
+import (
+	"encoding/json"
+	"github.com/HackIllinois/api/services/print/errors"
+	"github.com/HackIllinois/api/services/print/models"
+	"github.com/HackIllinois/api/services/print/service"
+	"github.com/gorilla/mux"
+	"github.com/justinas/alice"
+	"net/http"
+)
+
+func SetupController(route *mux.Route) {
+	router := route.Subrouter()
+
+	router.Handle("/", alice.New().ThenFunc(CreatePrintJob)).Methods("POST")
+}
+
+/*
+	Endpoint to send a print job to the sns topic
+*/
+func CreatePrintJob(w http.ResponseWriter, r *http.Request) {
+	var print_job models.PrintJob
+	json.NewDecoder(r.Body).Decode(&print_job)
+
+	_, err := service.PublishPrintJob(&print_job)
```

Comment:
Depending on what's in this variable being returned we might want to put some of that information into the response body.

Reply:
Just exposes the internal id of the Job that gets published to SNS